appveyor.yml

```diff
@@ -18,23 +18,52 @@ branches:
     - 0.11.x
     - 0.12.x
 
-matrix:
-  fast_finish: true
+environment:
+  matrix:
+    - TOXENV: check
+      TOXPYTHON: C:\Python36\python.exe
+      PYTHON_HOME: C:\Python36
+      PYTHON_VERSION: '3.6'
+      PYTHON_ARCH: '32'
+    - TOXENV: 'py35'
```

Comment:
Windows users tend to have more up-to-date Python installations, see for example https://github.com/ContinuumIO/anaconda-package-data:

![Screenshot_2019-04-21 anaconda_package_data_example(1)](https://user-images.githubusercontent.com/316517/56472399-5bb80200-645e-11e9-9512-27bceed7c3cb.png)

Therefore, I propose we remove Python 3.5 altogether, and even the 32 bit builds. The will probably slow us down for very little gain.